ibis/backends/postgres/registry.py

```diff
@@ -696,13 +667,7 @@ def _day_of_week_name(t, expr):
         ops.ExtractMillisecond: _millisecond,
         ops.DayOfWeekIndex: _day_of_week_index,
         ops.DayOfWeekName: _day_of_week_name,
-        ops.Sum: _reduction('sum'),
-        ops.Mean: _reduction('avg'),
```

Comment:
Are these implemented for the base class?

Reply:
Yep.